core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -19,97 +19,28 @@
 import static com.oath.oak.ValueUtils.ValueResult.*;
 
 public class Chunk<K, V> {
+    static final int NONE_NEXT = 0;    // an entry with NONE_NEXT as its next pointer, points to a null entry
 
     /*-------------- Constants --------------*/
 
-    /***
-     * This enum is used to access the different fields in each entry.
-     * The value associated with each entry signifies the offset of the field relative to the entry's beginning.
-     */
-    enum OFFSET {
```

Comment:
where is this OFFSET going? It also changes location in Anastasia's PR

Reply:
I am referring only to comments caused by my PR. Yes it is part of  "Anastasia's PR". It was moved to EntrySet.